### DISCUSSION TITLE
Welcome to Codigo Club Discussions!

### DISCUSSION THREAD

**Discussion by Lakorthus** [Lakorthus/codigo-club#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JeanFerreira91** [comment]
Now that we have deployed the Backend (API) of the blog, we need to make some small changes such as:

- [ ] The API's query response should be organised by 'created_on' field, with the most recent posts first.
- [ ] Once the previous step is done, we can just change the cards' loop to skip the first result.
- [ ] And finally, on the big card, we just query for the first item which should be the latest blog post.

Any other ideas/improvements?
